### ISSUE TITLE
Nextbots triggers errors.

### ISSUE THREAD

**Issue by CattoGamer** [lolleko/guesswho#46]
I'm receiving some errors wihile playing with my friends on the server. The error comes while colliding with the bots.

Version:  via Github
Server OS: Linux - Debian 7.11

> 
> [ERROR] gamemodes/guesswho/gamemode/shared.lua:77: attempt to index a nil value
> 
> 1. unknown - gamemodes/guesswho/gamemode/shared.lua:77

**Comment by lolleko** [comment]
Should be this line if you haven't changed anything in that file:

https://github.com/lolleko/guesswho/blob/master/gamemodes/guesswho/gamemode/shared.lua#L77

But I have no idea what in that line could be nil.

**Comment by CattoGamer** [comment]
I didn't changed anything, except on the Config and sh_animation.

EDIT:

Also the ConVars does exist as shown on line 77.

**Comment by CattoGamer** [comment]
Hey @lolleko 

It seems to be not the only one issue.
```
[ERROR] gamemodes/guesswho/gamemode/cl_hud.lua:215: attempt to index a nil value
  1. v - gamemodes/guesswho/gamemode/cl_hud.lua:215
   2. unknown - lua/includes/modules/hook.lua:84
```

After some research it seems to be caused by the ConVars, the client does not receive the ConVars.

**Comment by lolleko** [comment]
[ERROR] gamemodes/guesswho/gamemode/cl_hud.lua:329: Tried to use a NULL entity!
  1. GetClass - [C]:-1
   2. Run - gamemodes/guesswho/gamemode/cl_hud.lua:329
    3. unknown - gamemodes/base/gamemode/cl_init.lua:76

**Comment by CattoGamer** [comment]
I already found a solution for this, it seems that the Clients don't receive any ConVars from the Server.

A solution was to create Clientside ConVars, the settings in gueswho.txt are only working on single-player/p2p.

**Comment by lolleko** [comment]
Yup can confirm, I will add failsafes for all shared cvars in the next update.

**Comment by lolleko** [comment]
Should be fixed in  cfc6550
